scripts/dynamicHtml.js

```diff
@@ -11,8 +11,35 @@ Colours.forEach(function(value, key) {
 const colourtokens = Array.from(Colours.keys());
 
 
+function makeTabHeaderContent(uid, tabTitle = ""){
+  tabTitle = tabTitle || (PictureData[uid] && PictureData[uid]['fnName']) || "New Image";
+  let tabDirty = PictureData[uid] ? !!PictureData[uid]['configurationDirty'] : false;
+  let tabDirtyMarker = "";
+  if (tabDirty) {
+    tabDirtyMarker = `<small title="Has unsaved changes">&nbsp;*</small>`;
+  }
+  let tabSavedDataSize = getSavedFormDataSize(uid);
+  let tabSavedMarker = "";
+  if (tabSavedDataSize > 0) {
+    let originalWasResized = getSavedFormOriginalWasResized(uid);
+    let originalWasResizedTooltip = originalWasResized ? ` (stored resized because original was too big; re-upload to change area)` : "";
```

Comment:
We are not uploading.
```suggestion
    let originalWasResizedTooltip = originalWasResized ? ` (stored resized image because original was too big; re-select the image to change area)` : "";
```